src/main/java/lesson01/loopspracticing/PrimeNumbers.java

```diff
@@ -16,8 +17,8 @@ public static void main (String[] args) {
         System.out.println("List of prime numbers from 2 to 100 is " + PrimeNumbers.identifyPrimeNumbersArrayList());
     }
 
-    static ArrayList identifyPrimeNumbersArrayList () {
-        ArrayList primeNumbersList = new ArrayList();
+    static List<Integer> identifyPrimeNumbersArrayList () {
```

Comment:
It'd be better if the name started from "generate" and ended the "List" prefix. You return List not ArrayList.

Reply:
Method renamed.
Regarding return, lets discuss it on code review